### PR TITLE
Cross Platform Build and NPM Fix

### PR DESCRIPTION
**Pull request by davida26**
Cross platform build supporting: linux/arm64, linux/amd64, linux/amd64/v2, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6

Dockerfile modified to install npm and fix node error

Added build context to auto perform build ops

Fixes in image

Use docker-compose up to handle all operations

### CODE REVIEW

**Comment by nsheely** [comment]
Tested it out and looks good!

File: docker-compose.yml [hunk 1]
```diff
@@ -10,11 +10,13 @@ services:
       - "7474:7474"
       - "7687:7687"
     environment:
-      NEO4J_AUTH: neo4j/gradvek
-      NEO4J_dbms_security_auth__minimum__password__length: 7
+      NEO4J_AUTH: neo4j/gradvek1
```
Comment (nsheely) [inline docker-compose.yml:13]: We may want to update the password in application.properties and PerformanceTest.java to match this change

**Review by kevinpatel24 — DISMISSED** [review]
Looks good!

**Review by alyssabedard — APPROVED** [review]
looks good

**Review by nsheely — APPROVED** [review]

**Pull request merged**